pom.xml

```diff
@@ -266,15 +266,7 @@
           <configuration>
             <!-- excluding files that don't need a header update -->
             <excludes>
-              <exclude>src/main/java/com/hotels/bdp/waggledance/api/validation/constraint/TunnelRoute.java</exclude>
```

Comment:
These classes are no longer in this project so they can be removed from this exclusion section.